### PR TITLE
feat: add @nuxt/nitro module

### PR DESCRIPTION
**Pull request by KABBOUCHI**
## Types of changes

- [x] New Detector
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the README accordingly 
- [ ] I have added the icon of my detector in `icons/`

### CODE REVIEW

**Review by danielroe — COMMENTED** [review]
Would be good to mention in README too.

**Notes**:
- This currently only detects the Nitro Compat module.
- We will likely need to update this detector later on.

File: detectors/nuxt.modules.json [hunk 1]
```diff
@@ -394,5 +394,16 @@
     "detectors": {
       "js": "window.$nuxt?.$druxt"
     }
+  },
+  "@nuxt/nitro": {
+    "metas": {
+      "slug": "nuxt-nitro",
```
Comment (pi0) [inline detectors/nuxt.modules.json:400]:
```suggestion
      "slug": "nuxt-nitro-compat",
```

Detection method is for compat module (in nuxt3 with built-in nitro there is no env to detect)

File: detectors/nuxt.modules.json [hunk 2]
```diff
@@ -394,5 +394,16 @@
...
+      "name": "@nuxt/nitro",
+      "imgPath": null,
+      "url": "https://nitro.nuxtjs.org/"
```
Comment (pi0) [inline detectors/nuxt.modules.json:403]:
```suggestion
      "url": "https://nuxtjs.org/"
```

(Docs will move to v3.nuxtjs.org. nitro subdomain is no used)

**Review by pi0 — APPROVED** [review]
LGTM. Thanks 💚

**Comment by pi0** [comment]
We might also remove NITRO_PRESET and there would be no easy way to distinguish nitro from bridge or nuxt3.

**Comment by pi0** [comment]
I think best would be `X-POWERED-BY: 'nitro'` header. This way we can also detect static vs nitro handled routes :)

**Pull request closed without merging**